ts/interactions/conversations/unsendingInteractions.ts

```diff
@@ -341,14 +341,13 @@ const doDeleteSelectedMessages = async ({
 export async function deleteMessagesForX(
   messageIds: Array<string>,
   conversationId: string,
-  isPublic: boolean
+  enforceDeleteServerSide: boolean // should only be enforced for messages successfully sent on communities
```

Comment:
You can make this a doc string so we can see it on hover where the function is called.

```tsx
/** should only be enforced for messages successfully sent on communities */
enforceDeleteServerSide: boolean
```